### PR TITLE
Renaming sandbox

### PR DESCRIPTION
**Pull request by ivana-companieshouse**

### CODE REVIEW

File: groups/infrastructure/profiles/staging-eu-west-2/stagsbox/vars [hunk 1]
```diff
@@ -0,0 +1,5 @@
+environment = "stagsbox"
+aws_profile = "staging-eu-west-2"
+enable_container_insights = true
+
+ec2_key_pair_name = "ch-aws-staging"
```
Comment (bcullerton) [inline groups/infrastructure/profiles/staging-eu-west-2/stagsbox/vars:5]: Looks good, worth adding in live ahead of time?

**Review by bcullerton — COMMENTED** [review]
Probably also worth removing sandbox-staging env

File: groups/infrastructure/profiles/staging-eu-west-2/stagsbox/vars (hunk 1 above)
Comment (ivana-companieshouse) [reply groups/infrastructure/profiles/staging-eu-west-2/stagsbox/vars:5]: As long it's not triggered... ;-)

**Comment by ivana-companieshouse** [comment]
> Probably also worth removing sandbox-staging env

I expect it to be gone on merge. Apparently, squash and rename are not friends.

**Review by bcullerton — APPROVED** [review]

**Pull request merged**